cluster.do.yaml

```diff
@@ -15,16 +15,17 @@ addons:
 cluster:
   provider : digital_ocean
   master:
+    count: 2
     image: ubuntu-18-04-x64
     region: blr1
     size: s-2vcpu-4gb
   node_pools:
-    - count: 1
+    - count: 0
       image: ubuntu-18-04-x64
       region: blr1
       size: s-2vcpu-4gb
       name: medium
-    - count: 1
+    - count: 0
```

Comment:
can we leave it to the default value of 1

Reply:
Done